readthedocs/projects/models.py

```diff
@@ -697,29 +691,20 @@ def all_active_versions(self):
         """
         return self.versions.filter(active=True)
 
-    def supported_versions(self, flat=True):
+    def supported_versions(self):
         """
         Get the list of supported versions.
         Returns a list of version strings.
         """
         if not self.num_major or not self.num_minor or not self.num_point:
```

Comment:
These are the concept we can probably kill. The num_major etc are on every project, but was my failed scheme at "auto-setting" supported versions. We should probably just set sane defaults for these, or figure out another way of representing the concept of a "supported" version.